Simulator/conversions.py

```diff
@@ -118,25 +117,41 @@ def R(q):
 
 
 def quat(v):
+    """
+    Normalizes a quaternion (or any vector of four elements)
+    """
     assert len(v) == 4, f"can't make a quaternion out of: {v}"
     N = np.linalg.norm(v)
     assert N > 0, "Norm of input is 0. Can't make a quaternion out of this! Got {}".format(v)
     return v / N
 
 
 def conj(q):
+    """
+    Returns the conjugate quaternion 
+    """
     q = np.copy(q)
-    q[0] = -q[0]
+    q[1:4] = -q[1:4]
```

Comment:
Why is this better?

Reply:
The first one is wrong. They are not equivalent unless I'm missing something?

Reply:
They actually are equivalent! Unintuitively, `q == -q` when it comes to quaternions, so negating just the scalar or just the vector are both valid conjugations.